src/app/api/note/[noteId]/supports/route.ts

```diff
@@ -0,0 +1,54 @@
+import { NextRequest, NextResponse } from "next/server"
```

Comment:
```suggestion
import { type NextRequest, NextResponse } from "next/server"
```

`NextRequest` is only used as types.